src/scripts/src/choices.js

```diff
@@ -881,7 +881,7 @@ class Choices {
    * @private
    */
   _triggerChange(value) {
-    if (!value) {
+    if ((typeof value == 'undefined') || value == null) {
```

Comment:
You can lose the use of `typeof`. The following would work and avoid type coercion:

```js
if (value === undefined) || value === null) { ... }
```